src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -28,6 +28,10 @@ class Logger extends BaseLogger implements DebugLoggerInterface, ResetInterface
      */
     public function getLogs(/* Request $request = null */)
     {
+        if (\func_num_args() < 1 && __CLASS__ !== \get_class($this) && __CLASS__ !== (new \ReflectionMethod($this, __FUNCTION__))->getDeclaringClass()->getName()) {
```

Comment:
I would add `&& !$this instanceof \PHPUnit\Framework\MockObject\MockObject && !$this instanceof \Prophecy\Prophecy\ProphecySubjectInterface` to avoid triggering this when mocking the class with PHPUnit or Prophecy (the mock class will get its signature from the actual signature, so the argument won't be added until we add it for real, but that's fine as it will then be added automatically)

Reply:
that applies to all places of course